### PR TITLE
Add usage example to tf.nn.conv2d()

### PR DESCRIPTION
**Pull request by boronhub**

### CODE REVIEW

**Review by generationzcode — DISMISSED** [review]

**Comment by boronhub** [comment]
@mihaimaruseac please review this

**Comment by mihaimaruseac** [comment]
All builds failed and there are conflicts to be fixed

File: tensorflow/python/ops/nn_ops.py [hunk 1]
```diff
@@ -1883,6 +1883,14 @@ def conv2d_v2(input,  # pylint: disable=redefined-builtin
 
   Must have `strides[0] = strides[3] = 1`.  For the most common case of the same
   horizontal and vertices strides, `strides = [1, stride, stride, 1]`.
+  
+  Usage Example:
+  >>> kernel_in = np.array([
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/nn_ops.py]: Empty space above this line please.

**Review by mihaimaruseac — COMMENTED** [review]
Make sure you run the doctest locally as well as pylint.

**Comment by mihaimaruseac** [comment]
The conflict still needs to be resolved.

**Review by mihaimaruseac — COMMENTED** [review]
I expect pylint to complain due to spacing in the lists (for example `],[` is not right, there should be a space after `,`).

But let's run the tests and then you will have to fix the failures

**Comment by boronhub** [comment]
I did pylint, it did not give any complaints.

**Comment by mihaimaruseac** [comment]
Perfect. Then what's left is to fix the ubuntu cpu error.

**Comment by rthadur** [comment]
@boronhub please check the Ubuntu CPU error here 
https://source.cloud.google.com/results/invocations/bd11b04e-bca3-4721-950c-091d4e3022e2/targets/%2F%2Ftensorflow%2Ftools%2Fdocs:tf_doctest/tests

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by rthadur** [comment]
@mihaimaruseac there are some doctest failures , probably that needs to be fixed.

**Comment by rthadur** [comment]
@boronhub it is still failing doctest , can you please check and run the doc test locally as mentioned [here](https://www.tensorflow.org/community/contribute/docs_ref#test_on_your_local_machine)

**Comment by boronhub** [comment]
@mihaimaruseac builds are passing

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**